### PR TITLE
CTCP-3147: update api definition with page, count, and receivedUntil …

### PR DESCRIPTION
**Pull request by derekspalmer**
…params

### CODE REVIEW

File: resources/public/api/conf/2.0/application.yaml [hunk 1]
```diff
@@ -121,7 +121,9 @@ paths:
       description: >
         Get all departure declarations sent to the customs office of departure within the past 31 days.
 
-        You can filter for departure declarations updated after a specific date and time and/or submitted with a specified EORI.
+        You can filter for departure declarations updated after and/or before a specific date and time and/or submitted with a specified EORI.
+
+        For a multi-page query you can filter for a specific page along with/or not, the number of items per page.
```
Comment (drnaylor-oc) [inline resources/public/api/conf/2.0/application.yaml:126]: along with/or not?

Comment (derekspalmer) [reply resources/public/api/conf/2.0/application.yaml:126]: :+1:

Comment (fmcgrath) [reply resources/public/api/conf/2.0/application.yaml:126]: @derekspalmer - Would 'optionally' be a better choice than 'along with'? Also, should 'specified' be 'specific'?

Suggestion:
You can filter for departure declarations updated before and/or after a specific date and time, and optionally filter for departures with a specific EORI.

File: resources/public/api/conf/2.0/application.yaml [hunk 2]
```diff
@@ -862,7 +862,7 @@ paths:
       description: >
         Get all arrivals that have been updated within the past 31 days.
         
-        You can filter for arrival notifications updated after and/or before a specific date and time and/or submitted with a specified EORI.
+        You can filter for arrival notifications updated after and/or before a specific date and time along with/or not, a specified EORI.
```
Comment (fmcgrath) [inline resources/public/api/conf/2.0/application.yaml:866]: Same issues as above.

Suggestion:
You can filter for arrival notifications updated before and/or after a specific date and time, and optionally filter for arrival notifications with a specific EORI.

File: resources/public/api/conf/2.0/application.yaml [hunk 3]
```diff
@@ -316,10 +340,12 @@ paths:
       description: >
         Get all messages related to a specific departure declaration sent to the customs office of departure within the past 31 days.
 
-        You can filter for departure declarations received after a specific date and time.
+        You can filter for departure declarations received after and/or before a specific date and time.
```
Comment (fmcgrath) [inline resources/public/api/conf/2.0/application.yaml:344]: Should that be 'before and/or after'?

File: resources/public/api/conf/2.0/application.yaml [hunk 4]
```diff
@@ -1025,7 +1097,9 @@ paths:
       description: >
         Get all messages related to a specific arrival that have been received within the past 31 days.
         
-        You can filter for arrivals received after a specific date and time.
+        You can filter for arrival notifications received after and/or before a specific date and time.
```
Comment (fmcgrath) [inline resources/public/api/conf/2.0/application.yaml:1100]: Should that be 'before and/or after'?

File: resources/public/api/conf/2.0/application.yaml (hunk 2 above)
Comment (derekspalmer) [reply resources/public/api/conf/2.0/application.yaml:866]: Yeah, sorry thought I'd changed that!

**Review by fmcgrath — APPROVED** [review]
Looks good to me.

**Pull request merged**